### ISSUE TITLE
Transform to functional component

### ISSUE THREAD

**Issue by lemonshumhk01** [arunghosh/react-pin-input#216]
Hi author, may i know if this package will refactor to functional component base ?

**Comment by arunghosh** [comment]
@lemonshumhk01  Sorry. I was not able to understand the question. Can you elaborate?

**Comment by bolub** [comment]
@arunghosh , this project uses class components, and old react syntax, @lemonshumhk01 wanted to know if we'd be updating to the latest react component and maybe hooks syntax

**Comment by bolub** [comment]
If help is needed with that, i'd be able o

**Comment by arunghosh** [comment]
Yes. I would love to change this to functional component/hooks and Typescript. You help is most welcomed and appreciated.  Thank you :)
